modules/olm-creating-fb-index-image.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+ifdef::openshift-origin[]
+:registry-image: quay.io/openshift/origin-operator-registry:4.9.0
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.9
+endif::[]
+
+[id="olm-creating-fb-index-image_{context}"]
+= Creating a file-based index image
+
+You can create an index image that uses a plain text _file-based catalog_ format (JSON or YAML) instead of the deprecated SQLite database format. The `opm` CLI has new tooling that helps initialize an index in the file-based format, render new records into it, and then validate that the index is valid.
+
+.Prerequisites
+
+* `opm` version 1.18+
+* `podman` version 1.9.3+
+* A bundle image built and pushed to a registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2]
+
+.Procedure
+
+. Initialize an index for a file-based catalog:
+
+.. Create a directory for the index:
++
+[source,terminal]
+----
+$ mkdir <operator_name>-index
```

Comment:
Went in circles on whether to name this user-replaceable name `<operator_name>` vs `<package_name>`. "Package" is more accurate, I suppose, but thought maybe `<operator_name>` kept things a little simpler.

Reply:
I was actually _just_ having this same debate with @anik120. I suggested `package` because some bundles contain only API definitions.

Overall, I don't think it matters all that much, because folks splitting APIs from operator deployments are pretty advanced and would likely understand these docs based on context.